dts/xtensa/nxp/nxp_imx8m.dtsi

```diff
@@ -67,15 +69,17 @@
 			/* TODO: This INTID is just a dummy
```

Comment:
Should this TODO be removed? Or do we still not support UART interrupts with this PR?

Reply:
Not yet!
I've explained [here](https://github.com/zephyrproject-rtos/zephyr/pull/64637#discussion_r1397363785) why.
Briefly, after this PR is merged we can enabled UART interrupt by adding the irq_steer interrupt control in the dtsi + enabling some configs. But, will still have to wait a bit in order to not break SOF.